pyproject.toml

```diff
@@ -9,11 +9,16 @@ urls = { Repository = "https://github.com/robert-koch-institut/mex-model" }
 requires-python = "<3.13,>=3.11"
 dependencies = []
 optional-dependencies.dev = [
-    "black==24.8.0",
-    "ipdb==0.13.13",
-    "myst-parser==4.0.0",
-    "ruff==0.5.7",
-    "sphinx==7.4.7",
+    "black==24.8.0,<25",
+    "ipdb==0.13.1,<1",
```

Comment:
it should be greater than `>=` for black, ipdb and myst-parser as well. 

z.b: `"ipdb>=0.13.13,<1",`

Reply:
also check if `black` is still needed. It was removed from the template and some other repos: https://github.com/robert-koch-institut/mex-template/blob/a782ed7ea131e6d0cd67d4cc81975f2328228bcd/mex-%7B%7B%20cookiecutter.project_name%20%7D%7D/pyproject.toml#L11

Reply:
https://github.com/robert-koch-institut/mex-model/pull/107/commits/30678238ff52cda995e040f4dbecf8b2371e55fa